src/match-iterator.coffee

```diff
@@ -166,7 +182,11 @@ class ForwardsMultiLine
       match
 
 class BackwardsMultiLine
-  constructor: (@buffer, @regex, range, @chunkSize) ->
+  constructor: (@buffer, @regex, range, @options={}, @chunkSize) ->
+    if _.isNumber(@options)
+      @chunkSize = @options
+      @options = {}
```

Comment:
The three other classes don't have a `chunkSize` argument. I tried to keep the order of the common arguments the same (therefore `options` after `range`). The conditional is there to not break compatibility if any other code is calling this API.